loadclient/es_bulk_indexer.go

```diff
@@ -1,19 +1,16 @@
-package main
+package loadclient
```

Comment:
If the source code is indented to be kept private, then using `internal` as a package folder is a good choice. Only your `main.go` can reference something from `TOP_LEVEL_DIR/internal`. Take for example elasticsearch-operator.

Reply:
Added Internal
DONE